avogadro/qtplugins/bondcentrictool/CMakeLists.txt

```diff
@@ -19,4 +19,12 @@ avogadro_plugin(BondCentric
   "${bondcentrictool_rcs}"
 )
 
-target_link_libraries(BondCentric PRIVATE Avogadro::QtOpenGL)
+target_link_libraries(BondCentric
+  PRIVATE
+    Avogadro::Rendering
+    Avogadro::QtOpenGL)
+
+target_include_directories(BondCentric
```

Comment:
There's got to be a better way to do this with `cmake`. In theory, this is set up through the `function(avogadro_plugin` bits in `qtplugins/CMakeLists.txt`.

I'd guess that's a better thing to modify so this isn't needed in a bunch of plugin CMakeLists.txt

Reply:
Yeah, I think all those things aren't needed now that I've fixed the issue with `Avogadro::Rendering` being PRIVATE.

That was a fix I did 6+ months ago so the exact errors these fixes were solving are a little lost to the depths of my memory.

Reply:
Removed all the `target_include_directories` bits with the latest commit.